packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -379,13 +379,15 @@ const useSliceMachineActions = () => {
   const generateSliceScreenshot = (
     variationId: string,
     component: ComponentUI,
-    setData: (data: any) => void
+    setData: (data: any) => void,
+    screenWidth = "1200"
```

Comment:
I wouldn't set a default value here to avoid weird screenshot being taken if our code is fucked up.

This would be covering a code problem and user's screenshot would be fucked up.